src/bots/userPnlSettler.ts

```diff
@@ -156,21 +210,32 @@ export class UserPnlSettlerBot implements Bot {
 
 	public async startIntervalLoop(intervalMs?: number): Promise<void> {
 		logger.info(`${this.name} Bot started!`);
+		const negativeInterval = intervalMs || this.defaultIntervalMs;
+
 		if (this.runOnce) {
-			await this.trySettlePnl();
+			await this.trySettleNegativePnl();
 			await this.trySettleUsersWithNoPositions();
 		} else {
-			await this.trySettlePnl();
+			// Initial settlement
+			await this.trySettleNegativePnl();
 			await this.trySettleUsersWithNoPositions();
+
+			// Set up intervals
 			this.intervalIds.push(
-				setInterval(this.trySettlePnl.bind(this), intervalMs)
+				setInterval(this.trySettleNegativePnl.bind(this), negativeInterval)
 			);
 			this.intervalIds.push(
 				setInterval(
 					this.trySettleUsersWithNoPositions.bind(this),
 					EMPTY_USER_SETTLE_INTERVAL_MS
 				)
 			);
+			this.intervalIds.push(
+				setInterval(
+					this.trySettlePositivePnlForLowMargin.bind(this),
+					POSITIVE_PNL_SETTLE_INTERVAL_MS
+				)
+			);
```

Comment:
would it be better to just run the positive pnl loop immediately after the negative pnl loop instead of on its own timer?

Reply:
do you mean after x negative pnl loops? because we want positive settles to be on a longer interval. i think just fixed to hourly is ok